pimd/mtracebis.c

```diff
@@ -64,13 +63,14 @@ static void version(void)
 static void print_host(struct in_addr addr)
 {
 	struct hostent *h;
+	char buf[PREFIX_STRLEN];
 
 	h = gethostbyaddr(&addr, sizeof(addr), AF_INET);
 	if (h == NULL)
 		printf("?");
 	else
 		printf("%s", h->h_name);
-	printf(" (%s) ", inet_ntoa(addr));
+	printf(" (%s) ", inet_ntop(AF_INET, &addr, buf, sizeof(buf)));
```

Comment:
I think we can use `printfrr()` here with %pI4.

Reply:
Thanks - I was thinking to leave this particular module a bit more 'posix' and a bit less 'frr'? I don't know, maybe that's not really an issue?

Reply:
it's not an issue.  since the underlying pI4 uses inet_ntop right?

Reply:
yes, all good